src/main/java/ch/njol/skript/util/WeatherType.java

```diff
@@ -35,41 +36,38 @@
  * @author Peter Güttinger
  */
 public enum WeatherType {
-	
+
 	CLEAR, RAIN, THUNDER;
-	
+
 	String[] names;
-	
+
 	@Nullable
 	String adjective;
-	
+
 	final static Map<String, WeatherType> byName = new HashMap<>();
-	
-	private WeatherType(final String... names) {
+
+	WeatherType(final String... names) {

```

Comment:
why did you make it package-private?

Reply:
*Modifier 'private' is redundant for enum constructors* - IntelliJ, aka you can't create an instance of enums outside of the enum so the keyword is not needed